portfolio/src/main/webapp/load-contact.js

```diff
@@ -0,0 +1,55 @@
+const businessAccounts = [
+    { link: 'https://www.linkedin.com/in/ashley-granitto', icon: 'fab fa-linkedin-in', username: 'ashley-granitto'},
+    { link: 'mailto:aeg249@cornell.edu', icon: 'far fa-envelope', username: 'aeg249@cornell.edu'},
+    { link: 'https://github.com/ashleygranitto', icon: 'fab fa-github', username: '@ashleygranitto'},
+];
+
+const personalAccounts = [
+    { link: 'https://www.instagram.com/ashleygranitto/?hl=en', icon: 'fab fa-instagram', username: '@ashleygranitto'},
+    { link: 'mailto:granittoashley@gmail.com', icon: 'far fa-envelope', username: 'granittoashley@gmail.com'},
+    { link: 'https://www.youtube.com/channel/UCUWdAT3O2K4gJGyqkkBGxOA', icon: 'fab fa-youtube', username: 'Ashley Granitto'},
+];
+
+let current = 'business';
+
+// Switch from the business tab to the personal tab 
+function businessToPersonal() { 
+    if (current === 'business') { 
+        document.getElementById('business-body').style.display = 'none';
+        document.getElementById('personal-body').style.display = 'flex';
+    }
+    current = 'personal';
+}
+
+// Switch from the business tab to the personal tab 
+function personalToBusiness() {  
+    if (current === 'personal') { 
+        document.getElementById('business-body').style.display = 'flex';
+        document.getElementById('personal-body').style.display = 'none';
+    }
+    current = 'business';
+}
+
+// Append contact information to the business body
+function createBusinessBody() {
+    businessAccounts.forEach(function(account) {
+        document.getElementById('business-body').innerHTML +=  
+              '<article class="account-container"> \n' +
```

Comment:
You can do this with template strings

```javascript
const account = `<article class="account-container">
  <a class="account-icon" href="${account.link}">
    <i class="${account.icon}"><i>
  </a>
  <p class="account-username">${account.username}</p>
</article>`;
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Also something to look into is template libraries to divide your HTML into more manageable, reusable pieces and then use the library to combine them into a full page

Winnie suggested pug before
https://pugjs.org/api/getting-started.html

I've used handlebars in the past
https://handlebarsjs.com/

Reply:
Since this is mostly an exercise, I'd say going with template strings this time is fine. You'll be using template strings a lot. 
When you're working on your capstone, you may end up having to read up on these template libraries when figuring out how to structure your code.